python/ray/data/preprocessors/scaler.py

```diff
@@ -9,23 +9,64 @@
 
 
 class StandardScaler(Preprocessor):
-    """Scale values within columns based on mean and standard deviation.
+    r"""Scale and translate each column by its mean and standard deviation, respectively.
```

Comment:
Hmm should this be reversed, i.e. translate by mean and scale by standard deviation?

Reply:
Fixed